definitions/variable/energy/energy-final.yaml

```
@@ -564,6 +571,34 @@
     energy_use: all excl.heating, cooling, transportation
     sector: all
     unit: [EJ/yr, GWh]
+- Final Energy|Electricity|Other (excl. Heating, Industry, Cooling, Transport):
+    description: Final electricity consumption for non heating, non cooling, non e-transport, non industrial heat
+      uses,
+    energy_source: electricity
+    energy_use: all excl.heating, cooling, transportation
+    sector: all
+    unit: [EJ/yr, GWh]
+- Final Energy|Electricity|Other (excl. Heating,Industry, Cooling, Transport, Cooking):
```

Just to keep the convention:
```suggestion
- Final Energy|Electricity|Other (excl. Heating, Industry, Cooling, Transport, Cooking):
```